chia/rpc/wallet_rpc_api.py

```diff
@@ -541,7 +572,27 @@ async def create_new_wallet(self, request: Dict):
                 }
             else:  # undefined did_type
                 pass
-
+        elif request["wallet_type"] == "nft_wallet":
+            for wallet in self.service.wallet_state_manager.wallets.values():
+                if wallet.type() == WalletType.NFT:
+                    # TODO Modify this for NFT1
+                    log.info("NFT wallet already existed, skipping.")
+                    return {
```

Comment:
We might want to return the transaction here (or at least transaction ID). It's annoying to not have the transaction available, when you are waiting for it to confirm.

Reply:
not sure what you mean, no transaction is generated on nft wallet creation